src/recipe_dictionary.cpp

```diff
@@ -328,6 +337,71 @@ std::vector<const recipe *> recipe_subset::search(
 
     // prepare search
     switch( key ) {
+        case search_type::length: {
+            auto const error = [txt]( char const *, size_t /* offset */ ) {
+                throw math::runtime_error( _( string_format(
+                                                  "Failed to convert '%s' to length.\nValid examples:\n122 cm\n1101mm\n2   meter",
```

Comment:
These three spaces before `meter` are intended?

Reply:
Yes. In these examples I am trying to show off that spacing does not matter.

Reply:
I added a comment to the first occurrence of this.